style/amenity-points.mss

```diff
@@ -1847,7 +1847,7 @@
     text-line-spacing: @standard-line-spacing-size;
     text-fill: darken(@landform-color, 30%);
     [feature = 'natural_volcano'] { text-fill: #d40000; }
-    [feature = 'mountain_pass'] { text-fill: @mountain-pass; }
+    [feature = 'mountain_pass'] { text-fill: darken(@mountain-pass, 30%); }
```

Comment:
Please change this to use `@transportation-text` instead, so it will match the text for other `transportation-icon` colored symbols